stable/sumokube/templates/daemonset.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if and .Values.sumologic.accessId .Values.sumologic.accessKey -}}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  template:
+    metadata:
+      name: {{ .Release.Name }}
```

Comment:
Thanks for the review—replaced `{{ .Release.Name }`} with `{{ template "fullname" . }}` to match other charts.